Makefile-rpm-ostree.am

```diff
@@ -95,8 +95,10 @@ $(librpmostree_rust_path): Makefile $(LIBRPMOSTREE_RUST_SRCS)
 	  $(cargo) build --verbose $${frozen} $(CARGO_RELEASE_ARGS)
 EXTRA_DIST += $(LIBRPMOSTREE_RUST_SRCS) rust/Cargo.lock
 
-rpmostree-rust.h: $(top_srcdir)/rust/src/lib.rs $(top_srcdir)/rust/cbindgen.toml
-	cbindgen -c $(top_srcdir)/rust/cbindgen.toml $(top_srcdir)/rust -o $@.tmp && mv $@.tmp $@
+# See rust/build.rs - it uses cbindgen as part of the Rust build
+rpmostree-rust.h: $(librpmostree_rust_path)
+	$(AM_V_GEN) src=$$(find @abs_top_builddir@/target/@RUST_TARGET_SUBDIR@/ -name 'rpmostree-rust.h') && \
```

Comment:
I'll note for the permanent record this is a rather ugly hack; I debated having the `build.rs` write to `abs_top_srcdir` but then that won't be set if one just does `cd rust && cargo ...`.